vllm/distributed/device_communicators/shm_broadcast.py

```diff
@@ -108,8 +108,14 @@ def __init__(self,
             # created by the process. The following patch is a workaround.
             with patch("multiprocessing.resource_tracker.register",
                        lambda *args, **kwargs: None):
-                self.shared_memory = shared_memory.SharedMemory(name=name)
-            assert self.shared_memory.size == self.total_bytes_of_buffer
+                try:
+                    self.shared_memory = shared_memory.SharedMemory(name=name)
+                    assert self.shared_memory.size == self.total_bytes_of_buffer  # noqa
```

Comment:
qq: what is this `noqa` for?

Reply:
longer than 79 chars

Reply:
This is super minor, but then you can parentheses
```suggestion
                    assert (self.shared_memory.size == self.total_bytes_of_buffer)
```

Reply:
let's do it later, too busy rn :(